processor/src/lib.rs

```diff
@@ -333,6 +334,27 @@ where
         self.end_call_block(block)
     }
 
+    /// Executes the specified [Dyn] block.
+    #[inline(always)]
+    fn execute_dyn_block(
+        &mut self,
+        block: &Dyn,
+        cb_table: &CodeBlockTable,
+    ) -> Result<(), ExecutionError> {
+        // get target hash from the stack
+        let dyn_hash = self.stack.get_word(0);
+        self.start_dyn_block(block, dyn_hash)?;
+
+        // get function body from the code block table and execute it
+        let dyn_digest = dyn_hash.into();
+        let fn_body = cb_table
```

Comment:
minor nit: I think terminology like "function body" might be slightly confusing here.